experiment_api/install.sh

```diff
@@ -0,0 +1,8 @@
+# python 3
```

Comment:
This file looks out of place. We do need a requirements.txt somewhere, but I think an install script is not desired.